acquia/cli
# Fix #508: Install Drush 9 aliases to project dir

**Motivation**
Fixes #508

**Proposed changes**
- Install Drush 9 aliases to project dir instead of HOME/.drush. This requires you select a matching application, since the alias tarball includes _all_ project aliases, but in most cases you only want to install the one matching the current project.
- Refactor CommandBase to take advantage of features from other commands (such as current project dir). We should really move some of this stuff into traits or helper classes.

**Testing steps**

1. Follow the [contribution guide](https://github.com/acquia/cli/blob/master/CONTRIBUTING.md#building-and-testing) to set up your development environment.
2. Clear the kernel cache to pick up new and changed commands: `./bin/acli ckc`
3. Run `acli remote:aliases:download`
4. Verify that Drush 8 aliases are still downloaded as before
5. Verify that Drush 9 aliases require you to select a matching application, and install into the current directory.

**Merge requirements**
